cmake/kokkoskernels_tpls.cmake

```diff
@@ -424,7 +424,7 @@ KOKKOSKERNELS_ADD_TPL_OPTION(LAPACKE OFF "Whether to enable LAPACKE")
 # Set F77_BLAS_MANGLE macro based on Fortran-C interface (unless already set
```

Comment:
@e10harvey Do we need adding ```KOKKOSKERNELS_ADD_TPL_OPTION(ARMPL  OFF "Whether to enable ARMPL")``` before this line? And, do we need adding   ```KOKKOSKERNELS_IMPORT_TPL(ARMPL)``` at the end of this file? I am not sure if these are necessary because I don't know much about cmake.

Reply:
Yes, both are needed. `KOKKOSKERNELS_ADD_TPL_OPTION` adds `KokkosKernels_ENABLE_TPL_ARMPL:BOOL=OFF` to the cmake cache. `KOKKOSKERNELS_IMPORT_TPL(ARMPL)` ensures that the ArmPL module is searched for in the environment; the `FIND_PACKAGE` call executes the code in `cmake/Modules/FindTPLARMPL.cmake`.

Reply:
@e10harvey I think ```KOKKOSKERNELS_ADD_TPL_OPTION(ARMPL   OFF  "Whether to enable ARMPL")``` should be moved up to before setting checking and setting ```F77_BLAS_MANGLE``` to make it work.